apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-editor-nux/src/welcome-tour/style-tour.scss

```diff
@@ -41,3 +41,12 @@ $welcome-tour-card-media-extra-padding: 14%; // temporary value, to match the pa
 		}
 	}
 }
+
+// Overriding an existing core WP rule, because the ID selector is necessary
```

Comment:
just noting that we are not overriding a WP rule, we are just adding one more related to the `tour-kit.wpcom-tour-kit`